src/main/java/io/codelex/arithmetic/practice/NumberCheck.java

```diff
@@ -0,0 +1,21 @@
+package io.codelex.arithmetic.practice;
+
+public class NumberCheck {
+
+    public static void main(String[] arguments) {
+        System.out.println(checkNumbers(10, 15));
+        System.out.println(checkNumbers(6, 10));
+        System.out.println(checkNumbers(15, 4));
+        System.out.println(checkNumbers(25, 16));
+        System.out.println(checkNumbers(5, 15));
+    }
+    public static boolean checkNumbers(int number1,int number2) {
+    return number1 == 15 || number2 == 15 || number1 + number2 == 15 || Math.abs(number1 - number2) ==15;
```

Comment:
Arī trūkst formatting, turpmāk par to nekomentēšu - pārformatē visus uzdevumu failus. Formatēšnu var veikt arī mapei kopumā, var pagooglēt kā to izdarīt.